pkg/cmd/logs.go

```diff
@@ -417,7 +414,7 @@ func logPodGardenImproved(podName string) {
 	checkError(err)
 	pods, err := Client.CoreV1().Pods("garden").List(metav1.ListOptions{})
 	checkError(err)
-	project, err := getProjectForShoot()
+	project, err := getProjectObject()
```

Comment:
I just took a look at the `logPodGardenImproved` function and was wondering why the project object was needed.
It's only used here
```go
if strings.Contains(line, ("shoot=" + project.Name + "/" + target.Target[2].Name)) {
```

shouldn't it be sufficient to instead get the value from the target stack?

```go
if strings.Contains(line, ("shoot=" + getTargetName("project") + "/" + getTargetName("shoot"))) {
```

Note that I have also replaced `target.Target[2].Name` with getTargetName("shoot").

There should be a backlog item to replace all occurences of `target.Target[index].Name` with the corresponding `getTargetName(kind)` call

Reply:
done. as we move out getProjectObject. I am assume we can remove it out from getShootObject as well, and then remove getProjectObject.  let me if I need change back.

Reply:
and I never said you should remove the `getProjectObject` function, just calling it in this functions was not needed.